release/v0.50.x-celestia is missing the type `VestingData` which exists upstream

## Context

I'm trying to upgrade to cosmos/interchaintest. See https://github.com/celestiaorg/celestia-app/issues/5419

## Problem

v0.50.x-celestia is missing a type that exists on v0.50.x. The type is `VestingData`.

```
$ go test ./...
# github.com/strangelove-ventures/interchaintest/v8/chain/cosmos
/Users/rootulp/go/pkg/mod/github.com/cosmos/interchaintest/v8@v8.8.1/chain/cosmos/module_vesting.go:46:122: undefined: vestingcli.VestingData
FAIL    github.com/celestiaorg/celestia-app/test/interchain [build failed]
FAIL    github.com/celestiaorg/celestia-app/test/interchain/chainspec [build failed]
FAIL
```

## Proposal

Investigate why v0.50.x-celestia is missing the `VestingData` type. If it was accidental, bring it back.

--- a/x/auth/vesting/client/cli/tx.go
+++ b/x/auth/vesting/client/cli/tx.go
@@ -91,3 +91,13 @@ timestamp.`,
 
 	return cmd
 }
+
+type VestingData struct {
+	StartTime int64         `json:"start_time"`
+	Periods   []InputPeriod `json:"periods"`
+}
+
+type InputPeriod struct {
+	Coins  string `json:"coins"`
+	Length int64  `json:"length_seconds"`
+}

